dotCMS/src/main/java/com/dotmarketing/db/DockerSecretsUtil.java

```diff
@@ -0,0 +1,100 @@
+package com.dotmarketing.db;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Original code was taken from https://github.com/carsdotcom/docker-secrets-java/blob/master/src/main/java/com/cars/framework/secrets/DockerSecrets.java
+ * and modified to our needs (License type: Apache 2.0)
+ * @author github.com/carsdotcom
+ */
+public class DockerSecretsUtil {
+
+    private static final String SECRETS_DIR = "/run/secrets/";
+
+    public static Map<String, String> load() throws DotRuntimeException {
+        File secretsDir = new File(SECRETS_DIR);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'secretsDir' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)